### PR TITLE
generic fixes

### PR DESCRIPTION
**Pull request by PirosB3**
Other small fixes

### CODE REVIEW

File: urls.py [hunk 1]
```diff
@@ -21,6 +21,6 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
-    url(r'^cfp($|/)', ProposalCreate.as_view(success_url='/'), name="cfp"),
```
Comment (mzdaniel) [inline urls.py:24]: Did this slip by mistake?

Comment (PirosB3) [reply urls.py:24]: no, it was added below?

Comment (PirosB3) [reply urls.py:24]: oh, you mean the regex is not correct? I agree we should end it somewhere. Is it okay to be consistent and just not require an ending slash?

Comment (PirosB3) [reply urls.py:24]: took initiative with this, let me know if you prefer something else

Comment (mzdaniel) [reply urls.py:24]: Yes, lets do it that way for now.

**Comment by mzdaniel** [comment]
Thank you Daniel. PR rebased into master and live.

File: urls.py (hunk 1 above)
Comment (sesas) [reply urls.py:24]: I think the ending slash problem can be solved with: 
 http://stackoverflow.com/questions/1596552/django-urls-without-a-trailing-slash-do-not-redirect

**Pull request closed without merging**